packages/cerebral-website-tutorial/parts/01_get_started/src/components/HeaderButton/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
so every part is related to a chapter, hence the numbers, since the titles are not written in stone yet the foldernames could change